gallery/src/styles/NxFormSelect/NxFormSelectWidthsExample.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+const NxFormSelectWidthsExample = () =>
+  <div className="form-select-width-variants">
```

Comment:
For HTML examples that don't need to be react for one reason or another, we typically use plain HTML files for the examples rather than react

Reply:
> For HTML examples that don't need to be react for one reason or another, we typically use plain HTML files for the examples rather than react

Every other example in that directory is a `.tsx` file and a quick scan of other directories suggests that "typically" may be an exaggeration.

Reply:
> For HTML examples that don't need to be react for one reason or another, we typically use plain HTML files for the examples rather than react

Did it anyway for the practice and the lolz: https://github.com/sonatype/sonatype-react-shared-components/pull/586/commits/bf700677b7703673ce002987e37e3d11cb0722bf